### DISCUSSION TITLE
Window Panel similar to Vivaldi

### DISCUSSION THREAD

**Discussion by Marin-Kitagawa** [Floorp-Projects/Floorp#630] [Feature Request]
In Vivaldi, we can manage all the Workspaces, all the open tabs and the closed tabs using a Windows Panel. Firefox has Firefox View. We can integrate this Firefox View to the sidebar and this will act as the Window Panel. So, there will be no need for using the Tree Style Tab extension. We can directly use access all the tabs from the vertical tabs and even across the Workspaces.

**Comment by cantalupo555** [comment]
This feature would be interesting.
